### DISCUSSION TITLE
Search results based on user location

### DISCUSSION THREAD

**Discussion by Fenwickdf** [microsoft-search/pnp-modern-search#2524] [Q&A]
Hello, I am wondering if someone is able to advise if the following can be achieved through the configurations available in PNP Modern Search?  

As a user of SharePoint Search/ PNP GitHub I would like the search results to consider the end-users location/Region.  The results will display available content for the user's region and at the same time displaying content available across multiple SharePoint or other knowledge repositories.

**Comment by wobba** [comment]
If there is a property on the persons profile (UPA) matching a value per item you can use tokens in the query template.

https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#user-tokens

Something like: `{searchTerms} RefinableString100:{User.Department}`

**Reply by Fenwickdf** [reply]
Thank you for the prompt response. @wobba  :)   let me go and try this and get back to you.
